src/game/level.c

```diff
@@ -403,6 +429,20 @@ static bool Level_LoadItems(MYFILE *fp)
             }
 
             InitialiseItem(i);
+
+            // Calculate number of pickups in a level
+            for (int16_t j = 0; j < NUM_PICKUP_OBJ; j++) {
+                if (item->object_number == m_PickupObjs[j]) {
+                    m_LevelPickups++;
+                }
+            }
```

Comment:
What if we test if the draw routine is `DrawPickupItem`?

Reply:
That could work I think. Covers it all xcept for `PickUpScion4Collision` I think.